src/app/genome-tracker/_components/explore-summaries.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { FC, useState, useEffect, useMemo } from "react";
+import { useSpring, animated } from "@react-spring/web";
+import { Group } from "@visx/group";
+import { useRouter } from "next/navigation";
+import { VERNACULAR_GROUP_ICON, IconData } from "@/components/icon-bar";
+import { Stack, Text } from "@mantine/core";
+
+// Typed animated components
+const AnimatedDiv = animated("div");
+const AnimatedLabel = animated("div");
+const AnimatedImage = animated("image");
+
+const DISABLED_GROUPS = ["BACTERIA", "CYANOBACTERIA", "CONIFERS_AND_CYCADS"];
+
+export const ExploreSummaries = () => {
+  const [animals, producers] = useMemo(() => {
+    const all = Object.entries(VERNACULAR_GROUP_ICON)
+      .filter(([key]) => !DISABLED_GROUPS.includes(key))
+      .map(([, value]) => value);
+
+    return [all.filter((item) => item.group === 1), all.filter((item) => item.group === 2 || item.group === 3)];
+  }, [VERNACULAR_GROUP_ICON]);
+
+  return (
+    <Stack gap="sm">
+      <Text size="md" fw={600} c="midnight.9">
+        Animals
+      </Text>
+      <VernacularCarousel items={animals} />
+      <Text size="md" fw={600} c="midnight.9">
+        Plants and Microbial life
+      </Text>
+      <VernacularCarousel items={producers} reverse />
+    </Stack>
+  );
+};
+
+interface VernacularCarouselProps {
+  items: IconData[];
+  scrollSpeed?: number;
+  reverse?: boolean;
+  itemWidth?: number;
+  gapWidth?: number;
+  fadeWidth?: number;
+}
+
+export const VernacularCarousel: FC<VernacularCarouselProps> = ({
+  items,
+  scrollSpeed = 50,
+  reverse = false,
+  itemWidth = 130,
+  gapWidth = 18,
+  fadeWidth = 75,
+}) => {
+  const totalWidth = items.length * itemWidth + gapWidth * items.length;
+
+  const [scrollOffset, setScrollOffset] = useState<number>(0);
+  const [isHovered, setIsHovered] = useState<boolean>(false);
+
+  // Spring for controlling scroll speed
+  const [springs, api] = useSpring<{ speedFactor: number }>(() => ({
+    speedFactor: 1,
+    config: { tension: 120, friction: 20 },
+  }));
+
+  useEffect(() => {
+    api.start({ speedFactor: isHovered ? 0 : 1 });
+  }, [isHovered, api]);
+
+  useEffect(() => {
+    let frameId: number;
+    let last = performance.now();
+    const direction = reverse ? -1 : 1;
+
+    const step = (now: number) => {
+      const delta = now - last;
+      last = now;
+      const factor = springs.speedFactor.get() as number;
+      setScrollOffset((prev) => {
+        const next = prev + ((direction * delta * scrollSpeed) / 1000) * factor;
+        return ((next % totalWidth) + totalWidth) % totalWidth;
+      });
+      frameId = requestAnimationFrame(step);
+    };
+
+    frameId = requestAnimationFrame(step);
+    return () => cancelAnimationFrame(frameId);
+  }, [scrollSpeed, reverse, totalWidth, springs.speedFactor]);
```

Comment:
useEffect always makes me a bit dizzy with it's rules, and this one is a doozy. It probably isn't possible here but it might be easier to read if it was pulled out into a separate function so that we could focus on the rendering aspects of it. Again though, useEffect always tricks me so I could be wrong here